packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -161,10 +165,13 @@ export class SwapQuoter {
         this.orderbook = orderbook;
         this.expiryBufferMs = expiryBufferMs;
         this.permittedOrderFeeTypes = permittedOrderFeeTypes;
+        this.rfqtTakerApiKeyWhitelist = options.rfqtTakerApiKeyWhitelist || [];
+        this.rfqtMakerEndpoints = options.rfqtMakerEndpoints || [];
```

Comment:
`rfqtMakerEndpoints` seems to only be used to construct a `QuoteRequestor`, does it need to be set as a public accessor?

Reply:
Are we asserting they are `uris`?

Reply:
Addressed the `rfqtMakerEndpoints` bit in https://github.com/0xProject/0x-monorepo/pull/2541/commits/3c795d365d7680144c91234d0e2822d5156f9446.

We are not _directly_ asserting that they are URI's.  We're just passing them through to the quote requestor, and letting it fail silently if they're not valid.